WhiskyKit/Sources/WhiskyKit/Whisky/Bottle.swift

```diff
@@ -20,25 +20,14 @@ import Foundation
 import SwiftUI
 import os.log
 
-public class Bottle: Hashable, Identifiable, ObservableObject {
-    public static func == (lhs: Bottle, rhs: Bottle) -> Bool {
-        return lhs.url == rhs.url
-    }
-
-    public func hash(into hasher: inout Hasher) {
-        return hasher.combine(url)
-    }
-    public var id: URL {
-        self.url
-    }
-
+public class Bottle: ObservableObject, Equatable, Hashable, Identifiable, Comparable {
     public let url: URL
     private let metadataURL: URL
     @Published public var settings: BottleSettings {
         didSet { saveSettings() }
     }
     @Published public var programs: [Program] = []
-    public var inFlight: Bool = false
+    @Published public var inFlight: Bool = false
     public var isActive: Bool = false
```

Comment:
```suggestion
    public var isActive: Bool = false
```
Since you're at it, do you mind renaming this variable to `isAvailable`, `folderExists`, or something? Because it can be a bit confusing.